packages/vue-urql/src/utils.ts

```diff
@@ -26,6 +26,12 @@ const unwrap = <T>(maybeRef: MaybeRef<T>): T =>
     ? maybeRef.value
     : maybeRef;
 
+const _toString = Object.prototype.toString;
+const isPlainObject = (obj: any): boolean => {
```

Comment:
Ï think we might do better in re-using/copying our existing checks https://github.com/urql-graphql/urql/blob/main/packages/core/src/utils/variables.ts#L23-L24

Reply:
yes, that checks is for Files/Blobs only. I just think this might be overkill here, as finally urql/core is used with its internal validations.
What I am trying to do here - is just unwrap possible refs and pass everything as it is. That's why I found `isPlainObject()` way more generic: urql/core could be kind of "source of truth" for input validation